js/snake.js

```diff
@@ -90,6 +91,8 @@ class Game {
         //Check if next move is on some food
         if(newHead[0] == this.food[0] && newHead[1] == this.food[1]) {
             console.log("Yum yum !!!");
+            this.score.increment();

```

Comment:
What about using 
```javascript
this.snake.lenght()
```
to compute the score ?

Reply:
done